src/core/app.lua

```diff
@@ -236,6 +236,15 @@ function main (options)
       assert(not done, "You can not have both 'duration' and 'done'")
       done = lib.timer(options.duration * 1e9)
    end
+
+   local breathe = breathe
+   if options.measure_latency or options.measure_latency == nil then
+      local histogram = require('lib.histogram')
```

Comment:
Fine with me, I wasn't sure whether I wanted to propose this code to be "core", but if it seems "core" enough to you it's core enough for me :)  Sure, can re-use core.shm.root too.